src/main/java/in/projecteka/consentmanager/consent/ConsentArtefactRepository.java

```diff
@@ -180,6 +175,33 @@
                         }));
     }
 
+    @SneakyThrows
+    public Flux<ConsentExpiry> getConsentArtefacts(ConsentStatus consentStatus) {
+        return Flux.create(fluxSink -> dbClient.preparedQuery(SELECT_CONSENTS_TO_VALIDATE_EXPIRY)
+                .execute(Tuple.of(consentStatus.toString()),
+                        handler -> {
+                            if (handler.failed()) {
+                                fluxSink.error(new Exception("Failed to get GRANTED consents"));
```

Comment:
return immediately and remove else statement;

Reply:
done.